Information/CenterOfMass.FCMacro

```diff
@@ -647,10 +725,13 @@ class CenterofmassWidget(QtGui.QWidget):
                 if callable(getattr(s_, 'getGlobalPlacement', None)):
                     o_.Placement = s_.getGlobalPlacement()
                     # e.g. App::Link has no method getGlobalPlacement
-                elif float(f'{app.Version()[0]}.{app.Version()[1]}') < 0.22:
+                elif s_.TypeId != 'Assembly::JointGroup' and float(f'{app.Version()[0]}.{app.Version()[1]}') < 0.22 or s_.TypeId == 'App::Link':
                     if s_.InList:
                         for it in s_.InListRecursive:
-                            o_.Placement = o_.Placement.multiply(it.Placement)
+                            if it.TypeId != 'Assembly::JointGroup':
+                                o_.Placement = o_.Placement.multiply(it.Placement)
+                elif s_.TypeId == 'Assembly::JointGroup':
+                    continue
                 else:
```

Comment:
`UtilsAssembly.getGlobalPlacement(s_)` was a temporary solution for v0.22 but it doesn't work in 1.0 any more. Please add another `else` snippet to address latest FreeCAD using new `App.GeoFeature.getGlobalPlacementOf(targetObj, rootObj, subname)` FreeCAD/FreeCAD#16922 or your proposed solution but which won't be universally valid, I think.

Reply:
I there any documentation of how this `App.GeoFeature.getGlobalPlacementOf(targetObj, rootObj, subname)`  works?